domovoi.py

```diff
@@ -85,12 +98,16 @@ def run(self, processes_file):
 
 def main():
     parser = argparse.ArgumentParser()# Take in command line arguments
-    parser.add_argument('processes_file', help='text file of solar car processes') # Adds in a positional argument for the process_file
-    args = parser.parse_args()# Stores the arguments the user inputted
+    parser.add_argument('--primary', '-p', help='Open Domovoi in Primary Pi mode', action='store_true') # Adds a positional and optional argument for startup modes
+    args = parser.parse_args()
     os.makedirs("logs", exist_ok=True)
     logging.basicConfig(filename='logs/%s' % time.asctime(), format='%(asctime)s - %(levelname)s - %(message)s') # Makes a log of events in this session
     domovoi = Domovoi()
-    domovoi.run(args.processes_file)# Uses the arguments stored earlier to run the program. 
+    if(args.primary == True):
+        file = domovoi.ping_raspi()
+    else:
+        file = settings.SECONDARY_FILE;
+    domovoi.run(file)# Uses the arguments stored earlier to run the program.
```

Comment:
I know it wasn't from this PR. But it would make more sense to pass in start up parameters, like maybe a list of commands or something, instead of a settings file.

Reply:
So something like `./domovoi.py primary MAX_RESTART SLEEP_TIME SECONDARY_FILE PRIMARY_FILE HOST_IP` ?